### PR TITLE
Print style for About site & Study site

### PR DESCRIPTION
**Pull request by squiz-thoang**
# Description

Adding print style for footer, header and specific pages for About site & Study site
SQUIZ JIRA Ticket #UOMDEV-8

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] Make sure to do not repeat yourself (DRY)
- [x] Snapshots tested
- [ ] A11y tested
- [x] CSS utilises BEM naming convention and structure
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] Crossbrowser testing (IE11, Safari 8+, iOS 8.4+, Android 4.4+, Firefox ESR (v52.x), iPhone (4s,6), iPad 2, Galaxy s5) 
- [x] My changes generate no new warnings
- [ ] I have added tests (e2e and unit) that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Added the new component to the index.js (Vue and Lib) export

# Reviewer Checklist

- [x] Code is not repeated (DRY)
- [ ] ES6+ code is used where possible
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] CSS utilises BEM naming convention and structure
- [x] Snapshots tested against a copy of dev snapshots to check UI changes are intended
- [ ] A11y tested
- [ ] Crossbrowser tested in at least IE11

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 77b719aaec070b24cb54e82af42e73aa5046a236

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5f28b74a545f090008e0cc64

File: components/print.css [hunk 1]
```diff
@@ -0,0 +1,212 @@
+@media print {
+  * {
+    color: rgb(0, 0, 0) !important;
```
Comment (gepifanio) [inline components/print.css:3]: Hi Tiffany, please use the file `_vars.css` for colors

`--col-black-dark: 0, 0, 0;` 

`color: rgb(var(-col-black-dark));`

File: components/print.css [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+  a,
+  a:visited {
+    text-decoration: underline;
+  }
+
+  a[href]::after {
+    content: ' (' attr(href) ')';
+    word-break: break-all;
+    color: rgb(0, 0, 0);
```
Comment (gepifanio) [inline components/print.css:49]: --col-black-dark: 0, 0, 0;

File: components/print.css [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+  .ir a::after,
+  a[href^='javascript:']::after,
+  a[href^='#']::after {
+    content: '';
+  }
+
+  pre,
+  blockquote {
+    border: 1px solid rgb(247, 247, 247);
```
Comment (gepifanio) [inline components/print.css:76]: use variable

File: components/print.css [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    padding: 0 !important;
+  }
+
+  .btn,
+  .btn:visited {
+    border: none !important;
+  }
+
+  .table__row--inverted {
+    border-bottom: 1px solid rgb(171, 190, 213) !important;
```
Comment (gepifanio) [inline components/print.css:196]: use variables

File: components/print.css (hunk 1 above)
Comment (squiz-thoang) [reply components/print.css]: Hi @gepifanio,
The color is updated using var.

**Pull request merged**